qiskit_ibm_runtime/sampler.py

```diff
@@ -146,6 +153,40 @@ def __init__(
                     log levels are: ``DEBUG``, ``INFO``, ``WARNING``, ``ERROR``, and ``CRITICAL``.
                     The default level is ``WARNING``.
 
+            skip_transpilation: (DEPRECATED, use transpilation_settings.skip_transpilation instead)
+                Transpilation is skipped if set to True. False by default.
+
+            transpilation_settings: Qiskit transpiler settings. The transpilation process converts
+                operations in the circuit to those supported by the backend, swaps qubits with the
+                circuit to overcome limited qubit connectivity and some optimizations to reduce the
+                circuit's gate count where it can.
+
+                * skip_transpilation: Transpilation is skipped if set to True.
+                    False by default.
+
+                * optimization_settings:
+
+                    * level: How much optimization to perform on the circuits.
+                        Higher levels generate more optimized circuits,
+                        at the expense of longer transpilation times.
+                        * 0: no optimization
+                        * 1: light optimization
+                        * 2: heavy optimization
+                        * 3: even heavier optimization
+                        If ``None``, level 1 will be chosen as default.
```

Comment:
So this default `None` -> `1` is what Qiskit does, but it is not the optimal setting.  The primitives, in my mind at least, should by default give the user the best results on average that we know how provided that doing so is not overly expensive.  In this case `optimization_level=3` is by far a better default that level 1, and is not much more expensive than the former.  This is also important because most users never adjust any settings.